var/spack/repos/builtin/packages/unifycr/package.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class Unifycr(AutotoolsPackage):
+    """User level file system that enables applications to use node-local
+    storage as burst buffers for shared files. Supports scalable and efficient
+    aggregation of I/O bandwidth from burst buffers while having the same life
+    cycle as a batch-submitted job.
+    UnifyCR is designed to support common I/O workloads, including
+    checkpoint/restart. While primarily designed for N-N write/read, UnifyCR
+    compliments its functionality with the support for N-1 write/read."""
+
+    homepage = "https://github.com/LLNL/UnifyCR"
+    git      = "https://github.com/LLNL/UnifyCR.git"
+    url      = "https://github.com/LLNL/UnifyCR/archive/v0.1.1.tar.gz"
+
+    version('develop', branch='dev', preferred=True)
+    version('0.1.1', sha256='f0628f661d5eff67a55ba2bf254dc38636525c5e191d5f32b9e128294b8f8051')
+
+    variant('debug', default='False', 
+            description='Enable debug build options')
+    variant('hdf5', default='False',
+            description='Build with HDF5 (currently serial only)')
+    variant('numa', default='True',
+            description='Build with NUMA')
+    variant('verbose', default='False', 
```

Comment:
@CamStan 

If this just affects the messages that get generated during the build, then you could condition this on whether `spack.config.get('config:debug')` is true, vs. choosing a variant. One of the consequences of making this a variant is that if you `spack install unifycr+verbose` and `spack install unifycr~verbose` then two different versions of the package will get installed.

My guess is that you can always output verbose make messages since that all gets written to Spack log files, vs. dumped to the screen. If it affects the build speed though I could see you wanting to switch it on/off (which you could do by checking if Spack's debug mode is enabled).

Reply:
I see what you mean. It was primarily for the option of viewing the verbose output for an optimized build where we'd not necessarily want to build with debug enabled and also to keep from slowing the build speed by default by having it off normally. That was more from a developer perspective rather than user, so I'll just lump it in with the debug variant.

However, in this process I ran into another issue with our actual build. So this is not quite yet ready to merge.